Api/Controllers/OrdersController.cs

```diff
@@ -1,12 +1,40 @@
 using Microsoft.AspNetCore.Mvc;
 using Reservant.Api.Services;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Identity;
+using Microsoft.AspNetCore.Mvc;
+using Reservant.Api.Identity;
+using Reservant.Api.Models;
+using Reservant.Api.Validation;
+using Reservant.Api.Services;
+
 
 namespace Reservant.Api.Controllers;
 
 /// <summary>
 /// Managing orders
 /// </summary>
 [ApiController, Route("/orders")]
-public class OrdersController(OrderService orderService) : Controller
+public class OrdersController(OrderService orderService, UserManager<User> userManager) : Controller
 {
+    /// <summary>
+    /// Controller responsible for canceling orders
+    /// </summary>
+    /// <request code="404"> Order doesnt exist among unrealised orders </request>
+    /// <request code="200"> Succesful cancelation </request>
+    [HttpPost("{id:int}/cancel")]
+    [ProducesResponseType(200), ProducesResponseType(404)]
```

Comment:
Skoro będziemy wszystkie błędy zwracać za pomocą ValidationFaiulre, to nie będzie statusu 404, tylko 400